download_and_create_reference_datasets/v02/hail_scripts/write_combined_reference_data_ht.py

```diff
@@ -328,8 +66,7 @@ def get_ht(dataset, reference_genome):
     # 'select' and 'custom_select's to generate dict.
     select_fields = get_select_fields(config.get('select'), base_ht)
     if 'custom_select' in config:
-        custom_select_fn_str = config['custom_select']
-        select_fields = {**select_fields, **globals()[custom_select_fn_str](base_ht)}
+        select_fields = {**select_fields, **config['custom_select'](base_ht)}
```

Comment:
We just call the function now rather than looking up the name of the function in `globals()`.